examples/aws/cluster-template.yaml

```diff
@@ -229,6 +235,7 @@ apiVersion: addons.cluster.x-k8s.io/v1beta1
 kind: ClusterResourceSet
 metadata:
   name: crs-ccm
+  namespace: ${NAMESPACE}
```

Comment:
I think when specifying `-n <namespace>` in `kubectl apply`, namespace is being set for each resource from the list. It may however conflict with an already set value. Could this value be removed from templates instead?

Reply:
@Danil-Grigorev the namespace if there to align with `clusterctl` templates, where it's used to substitute namespace in `generate` command